app/registries/foreman/access_permissions.rb

```diff
@@ -757,4 +757,9 @@
     map.permission :edit_settings, { :settings => [:update],
                                      :'api/v2/settings' => [:update] }
   end
+
+  permission_set.security_block :table_preferences do |map|
+    map.permission :view_table_preferences, { 'api/v2/table_preferences': [:index, :show] }
+    map.permission :edit_table_preferences, { 'api/v2/table_preferences': [:create, :update, :destroy] }
```

Comment:
Is there a specific reason to group :create, :update and :destroy? We do not do that for any other resources but have a separate permission for each, so I am just wondering whether it makes sense.